portfolio/src/main/webapp/contact.html

```diff
@@ -27,5 +27,40 @@ <h2>You can also find me on LinkedIn and Github.</h2>
       <a href="https://www.linkedin.com/in/arianazhu/" class="fa fa-linkedin"></a>
       <a href="https://github.com/arianazhu" class="fa fa-github"></a>
     </div>
+
+    <br>
+    <div class="content comments">
+        <br><hr><br>
+        <h2>Sign the Guestbook!</h2>
+        <form action="/comments" method="POST">
+            <label for=:"comment-name">Name:</label><br>
+            <input type="text" name="comment-name" placeholder="Your name" required><br>
+            <br>
```

Comment:
Using line breaks for spacing isn't the best style.
You don't get control of the amount of space, and if you want to change it, you end up having to update this everywhere you have a similar objective. If this spacing is something you want to do for all submit buttons, then it would make sense to make this part of a "submit-button" CSS class style. If it is a more general use of white space between form input elements, you may wish instead to use `<div>` with a "form-input-space" or "form-input-vspace" class which will set the height of the spacing. Or to group the `<label>` and `<input>` inside a `<div >` with class form-group or input-group.

The only advantage you might see from this is that it should scale with font size. But there are other ways to accomplish that by setting the height style property with the unit "em". See https://www.w3schools.com/css/css_units.asp for more details on units.